File: 2015/j1.py
# code by savir singh

month = int(input())
day = int(input())

if month < 2:
    print("Before")
if month > 2:
    print("After")
if month == 2 and day > 18:
    print("After")
if month == 2 and day < 18:
    print("Before")

if month == 2 and day == 18:
    print("Special")

File: 2015/j1.cpp
// code by savir singh

#include <iostream>
#include <string>
using namespace std;

int main(){

    int month;
    int day;
    cin >> month;
    cin >> day;
    if(month<2) {
        cout << "Before";
    }
    if (month>2) {
        cout << "After";
    }
    if (month==2 && day>18) {
        cout << "After";
    }
    if (month==2 && day<18) {
        cout << "Before";
    }

    if (month==2 && day==18) {
        cout << "Special";
    }
    return 0;
}
